frontend/src/components/page-about/index/belief.js

```diff
@@ -1,104 +1,67 @@
 import * as React from "react"
-import arrowImage from "../../../images/chevron-right.png"
+import { Collapsible } from "../../collapsible"
 import {
     container,
     titleContainer,
     middleContainer,
-    arrow,
-    subHead,
-    box,
 } from "../../../css/belief.module.css"
 
+const BibleProp = "We believe the Bible, both the Old and New Testaments, is the only inspired, infallible, and authoritative Word of God. It is the supreme source of truth for Christian faith and living.";
+const GodProp = "We believe in one God, Creator of all things, infinitely perfect and eternally existing in three persons: Father, Son, and Holy Spirit.";
+const JesusProp = "We believe that Jesus Christ is true (fully) God and true (fully) man, having been conceived of the Holy Spirit and born of the virgin Mary. He died on the cross, the complete and final sacrifice for our sins according to the Scriptures. He arose bodily from the dead, and ascended into heaven where, at God’s right hand, He intercedes for His people and rules as Lord over all.";
+const HSProp = "We believe that the ministry of the Holy Spirit is to glorify the Lord Jesus Christ, and during this age to convict the world of sin. He also regenerates the believing sinner, indwelling, guiding, instructing, and empowering us for godly living and sacrifice.";
+const ManProp = "We believe that the man was created in the image of God, but fell into sin and is therefore destined for eternal death. Only through regeneration by the Holy Spirit can salvation and spiritual life be obtained.";
+const SalvationProp = "We believe that the shed blood of Jesus Christ and His resurrection provide the only grounds for justification and salvation for all who believe, and only those who receive Jesus Christ by faith are born of the Holy Spirit and thus become children of God.";
+const FEProp = "We believe in the visible, personal, glorious, imminent return of Jesus Christ, His bodily resurrection of the dead, the judgment of the just and the unjust, and the fulfillment of Christ’s Kingdom in the new heavens and the new earth. Then shall the eager expectation of creation be fulfilled and the whole earth shall proclaim the glory of God who makes all things new.";
+const churchProp = "We believe that the true Church is composed of all such persons who through saving faith in Jesus Christ have been regenerated by the Holy Spirit and are united together in the body of Christ. We believe that Jesus Christ is the Head of the Church, and that every local church has the right under Christ to decide and govern its own affairs.";
+const OrdProp = "We believe that water baptism and the Lord’s Supper are ordinances to be observed by the Church during the present age. They are, however, not to be regarded as means of salvation.";
+
 const Belief = () => (
     <div className = {container}>
```

Comment:
Optional: No need to change it now, but FYI for the future, Tailwind allows us to use its styles directly in HTML, which is a big part of its benefit. For example you can write this instead of having to write it in a separate css file and importing it:

```
<div className="pb-56 pt-31 h-full w-full">
```